src/libData/AccountStore/AccountStore.cpp

```diff
@@ -518,6 +521,8 @@ bool AccountStore::UpdateAccountsTemp(
   LOG_GENERAL(WARNING, "[AS] Finished Processing <"
                            << transaction.GetTranID() << "> ("
                            << (status ? "Successfully)" : "Failed)"));
+
+  AccountStoreSC::m_transactionLatency = ((double)r_timer_end(tpStart)/1000.0);
```

Comment:
better to use static_cast<>.

Reply:
Yes, that's the old c programmer coming out, he is still in there somewhere, again this functionality is temporary as traces and spans carry the timing information, this is just to get a metric onto the dashboard this week. I will mark the code.

Reply:
changed in code anyway.